### PR TITLE
Stories/performance measurement

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/views/style_guides/alerts.haml [hunk 1]
```diff
@@ -83,15 +83,15 @@
...
+          .icon-clock-o
         .c-tag__content
           %span Long-term Stayer
 
   .d-flex.mr-3.mb-5
     .c-tag.c-tag--info.client-flag
       .c-tag__wrapper
         .c-tag__icon
-          .icon-clock
+          .-o
```
Comment (awisema) [inline app/views/style_guides/alerts.haml]: `.icon-clock-o`?

File: config/initializers/marginalia.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 if Rails.env.development?
   Marginalia::Comment.components = [:application, :controller, :action, :line]
-  Marginalia::Comment.prepend_comment = true
+  # Marginalia::Comment.prepend_comment = true
```
Comment (awisema) [inline config/initializers/marginalia.rb]: Did you intend to turn this off in the repo?

File: drivers/performance_measurement/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+## PerformanceMeasurement README
+
+This README file should be used to explain the functionality of the driver.
```
Comment (awisema) [inline drivers/performance_measurement/README.md]: Yes, please :-)

File: drivers/performance_measurement/app/models/performance_measurement/report.rb [hunk 4]
```diff
@@ -0,0 +1,477 @@
...
+      Client.import!(
+        report_clients.values,
+        batch_size: 5_000,
+        on_duplicate_key_update: {
+          conflict_target: [:id],
+          columns: Client.attribute_names.map(&:to_sym),
+        },
+      )
+      Project.import!([:report_id, :project_id], involved_projects.map { |p_id| [id, p_id] }, batch_size: 5_000)
+      # binding.pry
```
Comment (awisema) [inline drivers/performance_measurement/app/models/performance_measurement/report.rb]: Cleanup

File: drivers/performance_measurement/app/models/performance_measurement/report.rb [hunk 5]
```diff
@@ -0,0 +1,477 @@
...
+        'Measure 5',
+        'Measure 7',
+      ]
+      # For now, we're using a fixed set of project types
+      options = filter.to_h
+      options[:project_type_codes] = [:es, :so, :sh, :th, :ph]
+      # Because we want data back for all projects in the CoC we need to run this as the System User who will have access to everything
+      options[:user_id] = User.setup_system_user.id
+
+      # TODO: remove this
```
Comment (awisema) [inline drivers/performance_measurement/app/models/performance_measurement/report.rb]: ?

**Review by awisema — CHANGES_REQUESTED** [review]

File: config/initializers/marginalia.rb (hunk 2 above)
Comment (eanders) [reply config/initializers/marginalia.rb]: Yes, I was testing what it looks like appended vs prepended.   At least in IRB/Pry it is way more readable appended.  Maybe I'll set that to `! Rails.env.development?`

**Review by awisema — APPROVED** [review]

**Pull request merged**